Cache/Main.lean

```diff
@@ -83,8 +102,22 @@ def main (args : List String) : IO Unit := do
   let goodCurl ← pure !curlArgs.contains (args.headD "") <||> validateCurl
   if leanTarArgs.contains (args.headD "") then validateLeanTar
   let get (args : List String) (force := false) (decompress := true) := do
-    let hashMap ← if args.isEmpty then pure hashMap else hashMemo.filterByFilePaths (toPaths args)
```

Comment:
Doesn't `hashMemo` already contain the dependency information?

Reply:
It only contains this for all files it has memoized this for. This is mathlib and all `extraRoots` (i.e., arguments passed to it). Running just `lake exe cache get` in a dependent project does not parse the files in the downstream project.

Reply:
Can we reuse the same code that populates this in miniget?